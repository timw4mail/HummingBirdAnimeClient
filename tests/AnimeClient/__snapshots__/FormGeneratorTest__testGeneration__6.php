<?php declare(strict_types=1);

return '<input id="access_token_expires" type="text" name="access_token_expires" readonly value="" />
';
