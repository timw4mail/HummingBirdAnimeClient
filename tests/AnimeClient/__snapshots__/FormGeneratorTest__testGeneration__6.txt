<input id="access_token_expires" type="text" name="access_token_expires" readonly value="0" />
